tests/foreman/installer/test_infoblox.py

```diff
@@ -0,0 +1,154 @@
+"""Tests for Infoblox Plugin
+
+@Requirement: Infoblox
+
+@CaseAutomation: Automated
+
+@CaseLevel: Acceptance
+
+@CaseComponent: CLI
```

Comment:
I guess  `@CaseCompoment`  must be other than **CLI** which stands for **hammer**, So maybe **@CaseComponent: INSTALLER** is the best option